resources/metadata/nexusUpload.yaml

```diff
@@ -39,16 +56,24 @@ spec:
         mandatory: true
         aliases:
           - name: nexus/url
-      - name: repository
+      - name: mavenRepository
         type: string
-        description: Name of the nexus repository.
+        description: Name of the nexus repository for Maven and MTA deployments. If this is not provided, Maven and MTA deployment is implicitly disabled.
         scope:
-        - PARAMETERS
-        - STAGES
-        - STEPS
-        mandatory: true
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        aliases:
+          - name: nexus/mavenRepository
```

Comment:
You should add an alias to nexus/repository to stay compatible to our users.

Reply:
I think a break now will be better in the long term as it will be confusing to new users, and I think we don't have much adoption here yet. Disagree?

Reply:
disagree, msg for example

Reply:
At least you should print a warning, but an alias will not harm. You can declare it even as deprecated.

Reply:
ok, will do an alias

Reply:
Our stageConfig will not work anymore